### PR TITLE
Temporary fix for missing tools in runners

### PR DESCRIPTION
**Pull request by yhaliaw**
Self-hosted runners spawned by the GitHub runner charm does not have some common tooling users of GitHub Actions expects. This is due to the virtual machine hosting the self-hosted runners are created with the standard Ubuntu images.

Support for custom images based on [GitHub Actions Runner Images](https://github.com/actions/runner-images) is planned for the future and should be the permanent fix for this issue. In the mean time, installing the missing tools after starting up a virtual machine serves as a temporary fix.

### CODE REVIEW

File: tox.ini [hunk 1]
```diff
@@ -93,7 +93,7 @@ commands =
 [testenv:static]
 description = Run static analysis tests
 deps =
-    bandit
+    bandit[toml]
     toml
```
Comment (arturo-seijas) [inline tox.ini:97]: You'll need to substitute toml for bandit[toml]

File: src/runner.py [hunk 2]
```diff
@@ -354,6 +354,57 @@ def _install_binary(self, binary: Path) -> None:
         if self.instance is None:
             return
 
+        # TEMP: Install common tools used in GitHub Actions. This will be removed once virtual
+        # machines are created from custom images/GitHub runner image.
+        logger.info("Setting up installation of docker...")
+        self._execute(["/usr/bin/apt", "update"])
+        self._execute(["/usr/bin/apt", "install", "ca-certificates", "curl", "gnupg", "lsb-release"])
```
Comment (arturo-seijas) [inline src/runner.py:361]: Aren't some of these already installed by default in Ubuntu? i.e. lsb-release?

**Review by arturo-seijas — COMMENTED** [review]
Some comments inline

**Review by jdkandersson — APPROVED** [review]

**Review by yanksyoon — APPROVED** [review]
LGTM!

**Pull request merged**